code/src/LikeHeart.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react'
+import { Emoji } from "./Emoji"
+import './LikeHeart.css'
+
+
+export const LikeHeart = (props) => {
```

Comment:
In case you didn't know - you don't need the `()` when you are only passing one argument
```suggestion
export const LikeHeart = props => {
```